src/content/reference/manifest.mdx

```diff
@@ -272,6 +272,8 @@ deploy:
 
 If you define an image in the okteto manifest, `okteto deploy` will always run in remote mode.  If you don't define an image, and you run in remote mode, okteto will use a [default container image](https://github.com/okteto/pipeline-runner).
 
+Before the okteto CLI synchronizes your files, it looks for a file named `.oktetodeployignore` in your local folder. If this file exists, the CLI will exclude any files and directories that match patterns in it. This helps to avoid unnecessarily synchronizing large or sensitive files and directories that are not used by the commands defined in the `deploy` section. More information on this topic [is available here](reference/file-synchronization.mdx#ignoring-files).
```

Comment:
@AdrianPedriza could you double check this?
My understanding is that `.oktetoignoredeploy` follows `.Dockerfileignore` syntax, not syncthing syntax.
Not sure if they are compatible

Reply:
can I get info on this one @AdrianPedriza @pchico83 ? would live to have this doc in place soon.

Reply:
thanks!